src/main/resources/features/theme 1/17 - scheme user.feature

```diff
@@ -25,26 +25,56 @@ Feature: 17 - Scheme user
     And I enter {month} in the field with id "external-month"
     And I enter {year} in the field with id "external-year"
     And I enter "10" in the "Number of days to acknowledge by" field
-    And I enter "Test Special Notice" in the "Message title" field
+    And I enter "Test SN 1" in the "Message title" field
```

Comment:
SN 1 - Can we better the test name SN 1 because it means nothing to testers/users